src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift

```diff
@@ -416,7 +432,12 @@ class ExposureDetectionViewModel: CountdownTimerDelegate {
 				isActive: true,
 				accessibilityIdentifier: AccessibilityIdentifiers.ExposureDetection.explanationTextHigh
 			)
-		])
+		]
+		let survayParameters = appConfiguration.eventDrivenUserSurveyParameters.common
```

Comment:
We should fetch the app config every time highRiskModel is requested. (see comment above about the sink)